pom.xml

```diff
@@ -906,6 +906,14 @@
           </licenseHeader>
         </configuration>
       </plugin>
+        <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <configuration>
+                <source>9</source>
```

Comment:
The library requires Java 8, we will not bump the requirement unless there is a good reason. Are you sure the changes require Java9?